buildmachine/install-load-tests.ps1

```diff
@@ -0,0 +1,100 @@
+#Requires -RunAsAdministrator
+
+[CmdletBinding()]
+param (
+    [Parameter()][string]$JMeterVersion = "5.4.1",
+    [Parameter()][string]$DownloadFolder = "C:\download",
```

Comment:
Toto nie je potrebné. Rozbalil by som to priamo do TEMP-u (`$env:TEMP`) a odtiaľ presunul tam kam treba.

Alebo možno ešte lepšie, parameter `$JMeterPath` brať ako _root_ zložku pre pod-zložku `jmeter`, tzn. jeho predvolená hodnota by bola iba `C:\tools`. Potom samotné stiahnutie ZIP-u spraviť do TEMP-u, rozbaliť ho priamo do `$JMeterPath`, čím vznikne zložka `C:\tools\apache-jmeter-5.4.1` a tú už len premenuješ na `jmeter` (žiadne kopírovanie hore-dolu).